waspc/src/Wasp/Generator/ServerGenerator.hs

```diff
@@ -127,7 +139,7 @@ genPackageJson spec waspDependencies = do
           object
             [ "depsChunk" .= N.getDependenciesPackageJsonEntry combinedDependencies,
               "devDepsChunk" .= N.getDevDependenciesPackageJsonEntry combinedDependencies,
-              "nodeVersionRange" .= show NodeVersion.nodeVersionRange,
+              "nodeVersionRange" .= (">=" <> show NodeVersion.oldestWaspSupportedNodeVersion),
```

Comment:
Why not keep a `nodeVersionRange` with the `>= 20.0.0`?

Also, I saw you changing the qualified import name from `NodeVersion` to `V` in some places, maybe you want to do it here as well 🤷

Reply:
You have `SV.Range [SV.gte oldestWaspSupportedNodeVersion]` and you can show it

Reply:
So I on purpose got rid of `nodeVersionRange`, because that indicates that we can put any range there, and then in the rest of Wasp code, we use that range -> but that means we don't really know if it is range bounded from both sides, or what.
So instead, I went with `oldestSupportedVersion`, which indicates quite clearly that we have oldest node version we support and that is it, which means range is bounded from just one side, and then I use it in different places directly with adding `>=` and similar. Result is that I repeat this `>=` in a couple of places, but in return I have this explicit knowledge of range being bounded from just one side, so I thought that is a better trade-off. I hope I explained this well, if not I would be happy to try to explain more.

Reply:
`NodeVersion` vs `V` -> I sometimes use different qualifiers, depends on the size / scope of the file where I use it. I think it is too hard to try to keep these always the same through the whole codebase. But I updated them now all to `NodeVersion`!